official/utils/misc/distribution_utils.py

```diff
@@ -68,6 +73,17 @@ def get_distribution_strategy(num_gpus,
     else:
       return tf.distribute.MirroredStrategy(devices=devices)
 
+  if distribute_strategy == "collective":
+    return tf.contrib.distribute.CollectiveAllReduceStrategy(
```

Comment:
oh no :(
need these to be in tf core soooon...

Reply:
Will update it shortly.